### ISSUE TITLE
From time to time, new builds are pushed with 0 sequences

### ISSUE THREAD

**Issue by corneliusroemer** [outbreak-info/outbreak.info#347]
For the second time, it seems that an erroneous build was pushed that caused 0 sequences to be present on the website.
![image](https://user-images.githubusercontent.com/25161793/116600952-32be8780-a92a-11eb-9704-fb4492a83ef6.png)

Such errors are unavoidable, but wouldn't it make sense to put a check in place so that the old version is kept until a new build completes with more than 0 sequences? Should be easy to implement and prevent total breakage of the website!

**Comment by flaneuse** [comment]
@corneliusroemer thanks for posting the issue.  We're aware that when the data updates, the API and therefore the Variant Reports go down for < 2 minutes.  It's not trivial to ensure a seamless data update but @juliamullen is iteratively improving our server-side codebase to minimize downtime.

**Comment by corneliusroemer** [comment]
Thanks! I see, so it's not a build mistake but an API transition issue. Knowing that it's just down for 2min means I'll just wait a bit next time without worrying.

**Comment by flaneuse** [comment]
Yep, it's expected behavior.  Usually the downtime is < 1 minute and we've never seen it last more than 3 minutes, which is why we've been focusing on adding additional functionality.  We're working towards a smoother transition and providing updates to the users that this is an expected update.

**Comment by corneliusroemer** [comment]
Wow, so I must have spent a lot of time on the site if I've encountered 0 sequences twice over the last few days ;)

**Comment by flaneuse** [comment]
I know, I was impressed you found the downtime!  Glad to hear you find the site useful.
